hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/downloader/CommonDownloaderProperties.java

```
@@ -114,6 +118,13 @@ public void init() {
             }
             sources.add(0, source);
         }
+
+        if (downloadRatio == null) {
+            // defaults to consensusRatio + 15%, but never higher than 100%
+            downloadRatio = BigDecimal.ONE.min(consensusRatio.add(new BigDecimal("0.15"), MATH_CONTEXT));
+        } else { // enforce that downloadRatio >= consensusRatio
+            downloadRatio = downloadRatio.max(consensusRatio);
```

prefer to throw an exception instead of capping the value set by user

---

Done.